src/Marker.svelte

```diff
@@ -35,6 +35,11 @@
 
 	  return () => marker.remove()
 	})
+	
+	afterUpdate(() => {
+	    // Update marker on map if the lat or lng has been changed
```

Comment:
I have a strict no comments rule across all of my codebases :) It's also pretty obvious what this code does.

Reply:
Done.. got rid of the hook itself.